doc/api/process.markdown

```diff
@@ -159,40 +159,60 @@ event:
       return reportToUser(JSON.pasre(res)); // note the typo
     }); // no `.catch` or `.then`
 
+Here is an example of a coding pattern that will also trigger
+`'unhandledRejection'`:
+
+    function SomeResource() {
+      // Initially set the loaded status to a rejected promise
+      this.loaded = Promise.reject(new Error('Resource not yet loaded!'));
+    }
+
+    var resource = new SomeResource();
+    // no .catch or .then on resource.loaded for at least a turn
+
+To deal with cases like this, which you may not want to track as developer
+error in the same way as other `'unhandledRejection'` events, you can either
```

Comment:
"which you may not want to track as developer error in the same way as other `'unhandledRejection'` events"

Sentence structure is a little weird here, also "errors".